cheri_compressed_cap_common.h

```diff
@@ -604,10 +701,14 @@ static bool _cc_N(fast_is_representable_new_addr)(bool sealed, _cc_addr_t base,
                                                   _cc_addr_t new_cursor) {
     (void)sealed;
     uint32_t bwidth = _CC_MANTISSA_WIDTH;
-    uint32_t highest_exp = (64 - bwidth + 2);
+
+    cursor &= _CC_CURSOR_MASK;
+    new_cursor &= _CC_CURSOR_MASK;
+
+    uint32_t highest_exp = (_CC_ADDR_WIDTH - bwidth + 2);
 
     _cc_length_t top = base + length;
-    // If top is 0xffff... we assume we meant it to be 1 << 64
+    // If top is 0xffff... we assume we meant it to be 1 << 64 // TODO: Should we? top is now 128 bit.
```

Comment:
That is a stale comment and should be deleted.

Reply:
Deleted